api/datadoghq/v1alpha2/datadogpodautoscaler_types.go

```diff
@@ -83,6 +87,11 @@ type DatadogPodAutoscalerSpec struct {
 	// +optional
 	Objectives []common.DatadogPodAutoscalerObjective `json:"objectives,omitempty"`
 
+	// HorizontalFallback defines how recommendations should be applied when in fallback mode.
+	// +optional
+	// +kubebuilder:default={}
+	HorizontalFallback *DatadogPodAutoscalerHorizontalFallbackPolicy `json:"horizontalFallback,omitempty"`
```

Comment:
Perhaps we should do:
```
fallback:
  horizontal:
```

Just in case we want to add more later, it's cleaner from the top level